docs/source/en/api/pipelines/kandinsky3.md

```diff
@@ -9,7 +9,25 @@ specific language governing permissions and limitations under the License.
 
 # Kandinsky 3
 
-TODO
+Kandinsky 3 is created by [Arkhipkin Vladimir](https://github.com/oriBetelgeuse), [Igor Pavlov](https://github.com/boomb0om), [Andrei Filatov](https://github.com/anvilarth), [Zein Shaheen](https://github.com/zeinsh).
+
+The description from it's Github page: 
+
+*Kandinsky 3.0 is an open-source text-to-image diffusion model built upon the Kandinsky2-x model family. In comparison to its predecessors, Kandinsky 3.0 incorporates more data and specifically related to Russian culture, which allows to generate pictures related to Russin culture. Furthermore, enhancements have been made to the text understanding and visual quality of the model, achieved by increasing the size of the text encoder and Diffusion U-Net models, respectively.*
```

Comment:
Let's instead talk a little bit more about the architecture here. I think mostly:
1. a very power text encoder: [FLAN-UL2](https://huggingface.co/google/flan-ul2)
2. special Unet blocks that's twice deeper but remain same parameter counts
3. movq decoder (same as kandinsky 2)